SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -185,77 +248,28 @@ void USpatialWorkerConnection::StartDevelopmentAuth(FString DevAuthToken)
 	}
 }
 
-void USpatialWorkerConnection::ConnectToReceptionist(bool bConnectAsClient)
+void USpatialWorkerConnection::ConnectToReceptionist()
```

Comment:
I'll trust that you've refactored `ConnectToReceptionist` and `ConnectToLocator` functions correctly (i.e. didn't check every line there, but I like and approve this change in general 👍 )

Reply:
Your trust is well placed